packages/state-transition/src/cache/epochCache.ts

```diff
@@ -571,7 +576,12 @@ export class EpochCache {
     this.proposersPrevEpoch = this.proposers;
 
     const currentProposerSeed = getSeed(state, this.currentShuffling.epoch, DOMAIN_BEACON_PROPOSER);
-    this.proposers = computeProposers(currentProposerSeed, this.currentShuffling, this.effectiveBalanceIncrements);
+    this.proposers = computeProposers(
+      currentProposerSeed,
+      this.currentShuffling,
+      this.effectiveBalanceIncrements,
+      currEpoch >= this.config.ELECTRA_FORK_EPOCH
```

Comment:
the better way would be parss fork and do forkseq check inside compute proposers